tyk-stack/README.md

```diff
@@ -61,21 +60,20 @@ This removes all the Kubernetes components associated with the chart and deletes
 helm upgrade tyk-stack tyk-helm/tyk-stack -n tyk --devel
 ```
 
+_Note: Upgrading from tyk-pro chart_
+
+If you were using `tyk-pro` chart for existing release, you cannot upgrade directly. Please modify the values.yaml base on your requirements and install using the new `tyk-stack` chart.
+
 ## Configuration
 
 To get all configurable options with detailed comments:
 
 ```bash
-helm show values tyk-helm/tyk-stack > values-stack.yaml --devel
+helm show values tyk-helm/tyk-stack > values-tyk-stack.yaml --devel
```

Comment:
Should it be `values-tyk-stack` or `values-stack`?

Reply:
let's use values-stack.yaml

Reply:
```suggestion
helm show values tyk-helm/tyk-stack > values-stack.yaml --devel
```